app/Jobs/InsertCSVJob.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Jobs;
+
+use App\CSV;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+use League\Csv\ResultSet;
+
+class InsertCSVJob implements ShouldQueue
+{
+    use Dispatchable, SerializesModels;
+
+    private $records;
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($records)
+    {
+        $this->records = $records;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $csvs = [];
+        dd($this->records);
+        foreach ($this->records as $record) {
+            array_push($csvs, $record);
+        }
```

Comment:
tìm cách sử dụng array_map

Reply:
thằng $this->records không phải thuộc kiểu array nên không xài được hàm array_map anh ơi